catalog/src/main/kotlin/org/openapitools/client/infrastructure/ApiClient.kt

```diff
@@ -0,0 +1,277 @@
+package org.openapitools.client.infrastructure
+
+import okhttp3.OkHttpClient
```

Comment:
Consider using Retrofit instead?

Reply:
If you look at the commit message, I have considered and instructed the generator to do to so 🙂

However, I do not intend to keep the API client as is anyways, as it doesn't fit in with our SDK ecosystem.